yt_dlp/extractor/vvvvid.py

```diff
@@ -78,11 +114,18 @@ class VVVVIDIE(InfoExtractor):
         'only_matching': True
     }]
     _conn_id = None
+    _default_user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.50 Safari/537.37'
+
+    @functools.cached_property
+    def _headers(self):
+        geo_verification_headers = self.geo_verification_headers()
+        http_headers = self.get_param('http_headers', {})
+        return {**http_headers, **geo_verification_headers, 'User-Agent': self._default_user_agent}
```

Comment:
We don't need to do anything with the params headers here. The headers that we pass to the `_download_webpage`/`_download_json`/etc methods already update the params headers, they don't overwrite them.
```suggestion

    @functools.cached_property
    def _headers(self):
        return {
            **self.geo_verification_headers(),
            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.50 Safari/537.37',
        }
```